MMA Removal Utility/README.md

```diff
@@ -0,0 +1,91 @@
+----------------------------------------------
+
+> The Azure Tenant Security Solution (AzTS) was created by the Core Services Engineering & Operations (CSEO) division at Microsoft, to help accelerate Microsoft IT's adoption of Azure. We have shared AzTS and its documentation with the community to provide guidance for rapidly scanning, deploying and operationalizing cloud resources, across the different stages of DevOps, while maintaining controls on security and governance.
+<br>AzTS is not an official Microsoft product – rather an attempt to share Microsoft CSEO's best practices with the community.
+
+# AzTS MMA Removal Utility
+
+-----------------------------------------------------------------
+## Overview 
+
+AzTS MMA Removal utility can be used to remove MMA extension/agent from Azure Virtual Machine (VMs) across subscriptions. Microsoft Monitoring Agent (MMA) will be retired on 31 August 2024 and users need to migarte to Azure Monitor agent (AMA). If you have already migrated all of your Virtual Machines (VMs) to start using AMA for monitoring, it is recommened to remove MMA agent to avoide dupliaction of logs.
+
+AzTS MMA Removal utility can help to remove MMA extension from VMs which already have AMA extension present at tenant scale in centralized manner instead of doing it per VM manullay.
+
+## How it works?
+
+AzTS MMA Removal utility works in two phases:
+1. Discovery Phase
+2. Removal Phase
+
+**1- Discovery Phase**
+
+In this phase, it will collect all Virtual Machines and their extensions inventory to identify, which VMs have both MMA and AMA agent. Users can validate such VMs, VMs having both MMA and AMA agent will be processed further to remova MMA extension in next 'Removal Phase'.
```

Comment:
Updated a bit, as now we are making removal process configurable where user can also remove even if only MMA agent is present.

In this phase, it will prepare the inventory of all Virtual Machines and their extensions to identify, which VMs have both MMA and AMA or only MMA agents installed. In next phase (i.e., 'Removal Phase'), user can choose to remove MMA extension from VMs having either both MMA and AMA agents or just MMA agent.